src/Symfony/Component/Cache/DoctrineProvider.php

```diff
@@ -13,6 +13,7 @@
 
 use Doctrine\Common\Cache\CacheProvider;
 use Psr\Cache\CacheItemPoolInterface;
+use Symfony\Abstraction\DependencyInjection\ResettableInterface as AbstractResettableInterface;
```

Comment:
This doesn't need aliasing here as use statements take precedence over same namespace classes?